NullPointerException when fetching the quotes through the API

When fetching the quotes through the API (e.g. https://loglist.xyz/api/quote/list?limit=10&page=1&filter=none&order=time), loglist shows an error page. Here's an excerpt from the logs:
```
play.api.UnexpectedException: Unexpected exception[NullPointerException: null]
        at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:355)
        at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:261)
        at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:429)
        at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:421)
        at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:453)
        at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
        at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
        at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
        at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
        at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
Caused by: java.lang.NullPointerException: null
        at io.circe.Printer$PrintingFolder.onString(Printer.scala:320)
        at io.circe.Printer$PrintingFolder.onString(Printer.scala:303)
        at io.circe.Json$JString.foldWith(Json.scala:359)
        at io.circe.JsonObject$LinkedHashMapJsonObject.appendToFolder(JsonObject.scala:343)
        at io.circe.Printer$PrintingFolder.onObject(Printer.scala:370)
        at io.circe.Printer$PrintingFolder.onObject(Printer.scala:303)
        at io.circe.Json$JObject.foldWith(Json.scala:421)
        at io.circe.Printer$PrintingFolder.onArray(Printer.scala:356)
        at io.circe.Printer$PrintingFolder.onArray(Printer.scala:303)
        at io.circe.Json$JArray.foldWith(Json.scala:390)
```

(which, interestingly enough, doesn't even refer to our code at all)

Need to investigate.

--- a/project/build.properties
+++ b/project/build.properties
@@ -1,1 +1,1 @@
-sbt.version=1.4.0
+sbt.version=1.3.13

--- a/scala/src/main/scala/ru/org/codingteam/loglist/dto/QuoteDTO.scala
+++ b/scala/src/main/scala/ru/org/codingteam/loglist/dto/QuoteDTO.scala
@@ -1,3 +1,3 @@
 package ru.org.codingteam.loglist.dto
 
-case class QuoteDTO(id: Long, source: String, sourceUrl: String, time: Long, content: String, rating: Int)
+case class QuoteDTO(id: Long, source: String, sourceUrl: Option[String], time: Long, content: String, rating: Int)

--- a/scalajvm/app/controllers/api/Quotes.scala
+++ b/scalajvm/app/controllers/api/Quotes.scala
@@ -50,7 +50,7 @@ class Quotes @Inject()(implicit cc: ControllerComponents, configuration: Configu
     }
 
   private def buildQuoteDto(quote: Quote): QuoteDTO =
-    QuoteDTO(quote.id, quote.source, quote.sourceUrl.orNull, quote.time.toInstant.toEpochMilli, quote.content.getOrElse(""), quote.rating)
+    QuoteDTO(quote.id, quote.source, quote.sourceUrl, quote.time.toInstant.toEpochMilli, quote.content.getOrElse(""), quote.rating)
 
   private def json(text: String) = Ok(text).as("application/json; charset=utf-8")
 }

